README

```diff
@@ -88,6 +88,7 @@ Example:
  XLIO DETAILS: TCP max syn rate               0 (no limit)               [XLIO_TCP_MAX_SYN_RATE]
  XLIO DETAILS: Zerocopy Mem Bufs              200000                     [XLIO_ZC_BUFS]
  XLIO DETAILS: Zerocopy Cache Threshold       10 GB                      [XLIO_ZC_CACHE_THRESHOLD]
+ XLIO DETAILS: Tx Allowed Task Pool Expands   0                          [XLIO_TX_ALLOWED_TASK_POOL_EXPANDS]
```

Comment:
Not sure customer will understand the functionality from the name.
I think its implementation detail "task pool expansion",
in my opinion it should be in pkt size or memory size, and we should convert it internally.
what do you think?

Reply:
You're super right. changed it - wdyt?